<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1156</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1141</width>
     <height>631</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Schedule">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QGroupBox" name="singleDispatch">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>361</width>
       <height>371</height>
      </rect>
     </property>
     <property name="title">
      <string>Single Dispatch</string>
     </property>
     <widget class="QComboBox" name="chooseStationCombo">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>34</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="stationsTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>341</width>
        <height>192</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Stations</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="chooseStationLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Choose Stations:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addDestinationButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>341</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="dispatchButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>341</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Dispatch</string>
      </property>
     </widget>
    </widget>
    <widget class="QTableWidget" name="trainTable">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>20</y>
       <width>301</width>
       <height>561</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Train ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Block</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Next Stop</string>
      </property>
     </column>
    </widget>
    <widget class="QGroupBox" name="autoDispatch">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>490</y>
       <width>361</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Automatic Dispatch</string>
     </property>
     <widget class="QPushButton" name="openFile">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>341</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Open File</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="redLineButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Red Line</string>
     </property>
    </widget>
    <widget class="QPushButton" name="greenLineButton">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Green Line</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="editDispatch">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>100</y>
       <width>361</width>
       <height>481</height>
      </rect>
     </property>
     <property name="title">
      <string>Edit Dispatch</string>
     </property>
     <widget class="QPushButton" name="addButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>341</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QLabel" name="addStationLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Stations:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="addStation">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>84</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="removeButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>226</y>
        <width>341</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove</string>
      </property>
     </widget>
     <widget class="QLabel" name="removeStationLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>186</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove Stations:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="removeStation">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>180</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="stationsTable_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>341</width>
        <height>201</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Stations</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="chooseTrainLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>36</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Choose Train:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="chooseTrain">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="throughputLabel">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>50</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Throughput:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="throughPutValue">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>50</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>--</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="currentLineValue">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>10</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>--</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="currentLineLabel">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>10</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Current Line:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Maintenance">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
